assets/js/admin-external-connection.js

```diff
@@ -337,6 +341,18 @@ function checkConnections() {
 						}`,
 						'polite'
 					);
+				} else if ( response.data.errors.no_distributor ) {
+					endpointResult.innerText = __(
+						'Distributor not installed on remote site.',
+						'distributor'
+					);
+					speak(
+						__(
+							'Distributor not installed on remote site.',
+							'distributor'
+						),
+						'polite'
+					);
 				} else {
```

Comment:
This is to make it a little clearer that the site has Distributor but the account has low or no permissions.

```suggestion
				} else if (
					'no' === response.data.is_authenticated ||
					response.data.errors.no_types
				) {
					endpointResult.innerText = __(
						'Authentication failed due to insufficient or invalid credentials.',
						'distributor'
					);
					speak(
						__(
							'Authentication failed due to insufficient or invalid credentials.',
							'distributor'
						),
						'polite'
					);
				} else {
```